aurblobs/container.py

```diff
@@ -0,0 +1,84 @@
+import os
+import sys
+
+import click
+import docker
+from docker.errors import BuildError, APIError, ImageNotFound
+
+from .constants import DOCKER_IMAGE, DOCKER_BASE_IMAGE
+
+
+def need_rebuild():
+    client = docker.from_env()
+
+    # base image does not exist
+    try:
+        baseimage = client.images.get(DOCKER_BASE_IMAGE)
+    except ImageNotFound:
+        click.echo(
+            'Build container rebuild necessary: base layer missing'
+        )
+        return True
+
+    # base image outdated
+    baseimage_new = client.images.pull(DOCKER_BASE_IMAGE)
+    if baseimage.id != baseimage_new.id:
+        click.echo(
+            'Build container rebuild necessary: base layer update available'
+        )
+        return True
+
+    # build image does not exist
+    try:
+        image = client.images.get(DOCKER_IMAGE)
+    except ImageNotFound:
+        click.echo(
+            'Build container rebuild required: image is missing'
```

Comment:
"Image for build container not found. Need to build it" ?

Because "rebuild" would imply that we already have it but need to build it again